OpenXmlPowerTools/NugetPush.bat

```diff
@@ -0,0 +1,2 @@
+nuget push C:\Work\github\Open-Xml-PowerTools\OpenXmlPowerTools\bin\Release\*.nupkg -ApiKey 123.123a -Source http://nuget.cefcfco.com 
+pause 
```

Comment:
guess this shouldn't be here

Reply:
you`re right, this is build script for my local nuget server. I will remove it soon.